### PR TITLE
Chore/give panorama preview a width property

### PR DESCRIPTION
**Pull request by Jeffrey-Zutt**
- Give panorama a width and radius property
- Fix issues with newly created cases

### CODE REVIEW

File: src/app/features/cases/components/organisms/BagMap/BagMap.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Wrapper = styled.div`
 const BagMap: React.FC<Props> = ({ bagId, zoom = 20 }) => {
   const { data } = useBAG(bagId)
 
-  if (data === undefined) {
+  if (data === undefined || data.results[0] === undefined) {
```
Comment (gilleswittenberg) [inline src/app/features/cases/components/organisms/BagMap/BagMap.tsx:32]: `data?.results[0] === undefined`?

File: src/app/state/rest/index.ts [hunk 2]
```diff
@@ -85,11 +85,11 @@ export const useBAG = (bagId: string, options?: Options) => {
   })
 }
 
-export const usePanorama = (lat?: number, lon?: number, options?: Options) => {
+export const usePanorama = (lat?: number, lon?: number, width?: number, radius?: number, options?: Options) => {
   const handleError = useErrorHandler()
   return useApiRequest<{ url: string }>({
     ...options,
-    url: `https://api.data.amsterdam.nl/panorama/thumbnail/?lat=${ lat }&lon=${ lon }&width=438&radius=180`,
+    url: `https://api.data.amsterdam.nl/panorama/thumbnail/?lat=${ lat }&lon=${ lon }&width=${ width }&radius=${ radius }`,
```
Comment (gilleswittenberg) [inline src/app/state/rest/index.ts:92]: Maybe a `buildQueryString(params: Record<string, any>): string` would be nice

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/features/cases/components/organisms/BagMap/BagMap.tsx (hunk 1 above)
Comment (Jeffrey-Zutt) [reply src/app/features/cases/components/organisms/BagMap/BagMap.tsx:32]: Thanks, will change.

File: src/app/state/rest/index.ts (hunk 2 above)
Comment (Jeffrey-Zutt) [reply src/app/state/rest/index.ts:92]: Will make a seperate PR for it. Thanks

**Pull request merged**